src/js/Content/Features/Community/App/FCommunityAppPageWishlist.js

```diff
@@ -61,12 +50,11 @@ export default class FCommunityAppPageWishlist extends Feature {
             const action = wishlisted ? "wishlist.remove" : "wishlist.add";
 
             try {
-                await Background.action(action, that.appid);
+                await Background.action(action, appid);
 
                 successBtn.style.display = wishlisted ? "none" : "";
                 addBtn.style.display = wishlisted ? "" : "none";
 
-                DynamicStore.clear();
```

Comment:
This is already called in Background.action ?

Reply:
You're right